lib/src/easy_applovin/easy_applovin_banner_ad.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:applovin_max/applovin_max.dart';
+import 'package:easy_ads_flutter/src/easy_ad_base.dart';
+import 'package:easy_ads_flutter/src/enums/ad_network.dart';
+import 'package:easy_ads_flutter/src/enums/ad_unit_type.dart';
+
+class EasyApplovinBannerAd extends EasyAdBase {
+  EasyApplovinBannerAd(String adUnitId) : super(adUnitId);
+
+  @override
+  AdUnitType get adUnitType => AdUnitType.banner;
+  @override
+  AdNetwork get adNetwork => AdNetwork.appLovin;
+
+  @override
+  void dispose() {}
+
+  @override
+  bool get isAdLoaded => false;
+
+  @override
+  Future<void> load() async {}
```

Comment:
why not implementing these like we're doing in admob banners

Reply:
because applovin does not provide anything for the banner. It handles all of its implementation itself natively, so we does not need ay implementation of this.

you can see the docs [here](https://dash.applovin.com/documentation/mediation/flutter/getting-started/banners)

Reply:
Yeah but I guess we are not creating banner ad anywhere like written in docs

```
function initializeBannerAds()
{
    // Banners are automatically sized to 320x50 on phones and 728x90 on tablets
    AppLovinMAX.createBanner(_banner_ad_unit_id, AdViewPosition.bottomCenter);
}
```

Reply:
okay best. I hope you've tested it on both android and iOS.
btw thanks for the information.

Reply:
if everything is good to go, please resolve this convo. I'll approve the pr.